core/vm/gas_table.go

```diff
@@ -256,27 +256,27 @@ func makeGasLog(n uint64) gasFunc {
 			return multigas.ZeroGas(), 0, ErrGasUintOverflow
 		}
 
-		multiGas, gas, err := memoryGasCost(mem, memorySize)
+		multiGas, _, err := memoryGasCost(mem, memorySize)
 		if err != nil {
 			return multigas.ZeroGas(), 0, err
 		}
 
-		// TODO(NIT-3484): Update multi dimensional gas here
-		if gas, overflow = math.SafeAdd(gas, params.LogGas); overflow {
+		if overflow = multiGas.SafeIncrement(multigas.ResourceKindComputation, params.LogGas); overflow {
```

Comment:
My assumption: `HistoryGrowth := requestedSize * params.LogDataGas (8)`, others cost values are computation. Not sure about `bloomHistoryGrowthCost`.

Based on:
https://github.com/OffchainLabs/go-ethereum/blob/496de0b167b23ae2710059abd75f0ef18830519a/eth/tracers/native/gas_dimension_calc.go#L570

@relyt29

Reply:
Fixed